projects/core/src/product/event/product.events.ts

```diff
@@ -11,21 +11,21 @@ export class FacetChangedEvent extends CxEvent {
   /**
    * Facet code
    */
-  code: string;
+  code?: string;
```

Comment:
Would it have been possible to change the code that uses this interface instead of making all fields optional?

Reply:
So you're saying it's fine we make such fields optional?

Reply:
`facetCode` is optional in `Breadcrumb`. This `code` is `facadeCode`. So, it should be the same type as `facadeCode`. If we don't want it optional, we can set it to `''`.